packages/api/.env.example

```diff
@@ -15,3 +15,10 @@ DISABLE_EXTERNAL_API=false
 DATABASE_STATEMENT_TIMEOUT_MS=90000
 CONTRACT_VERIFICATION_API_URL=http://127.0.0.1:3070
 NETWORK_NAME=testnet-sepolia
+BASE_TOKEN_SYMBOL=wETH
```

Comment:
Please set it to ETH here so that if I copy the example env ETH is set by default.